src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -613,6 +633,11 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 "BCP110",
                 $"The type \"{type}\" does not contain function \"{name}\". Did you mean \"{suggestedName}\"?",
                 new CodeFix($"Change \"{name}\" to \"{suggestedName}\"", true, CodeManipulator.Replace(TextSpan, suggestedName)));
+
+            public ErrorDiagnostic ModulePathContainsAsciiControlChars() => new ErrorDiagnostic(
+                TextSpan,
+                "BCP111",
+                $"The specified module path contains invalid ASCII control code characters.");

```

Comment:
>ASCII [](start = 61, length = 5)

The ASCII part seems a little fishy to me...

Reply:
Fair. Are you happy with the message if I omit it? e.g. "The specified module path contains invalid control code characters."

Reply:
Yeah I think that works.

---
In reply to: [516198538](https://github.com/Azure/bicep/pull/766#discussion_r516198538) [](ancestors = 516198538)